src/Home/Home.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import DisplayBackgroundImage from '../images/DisplayBackgroundImage';
 
-function Home() {
+interface getUserProp{
```

Comment:
the convention is have the props name be ComponentNameProps - so in this case it would be `HomeProps` please can you change this to match that